### DISCUSSION TITLE
ToolDiy/platform/excalidraw/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#28] [Announcements]
# ToolDiy/platform/excalidraw/

https://cargo-youth.github.io/ToolDiy/platform/excalidraw/

**Comment by storage-db** [comment]
好棒的绘制软件

**Comment by Flurry-L** [comment]
这个真的好棒啊！

**Comment by Mc-GrowlR** [comment]
此外， 笔记软件 `Obsidian` 也可以安装 Excalidraw 插件来使用它

**Reply by storage-db** [reply]
蟀大佬来个star吧嘿嘿
